artiq/gateware/eem.py

```diff
@@ -125,17 +126,17 @@ def io_qspi(eem0, eem1, iostandard="LVDS_25"):
             ),
         ]
         ttls = [(6, eem0, "io_update"),
-                (7, eem0, "dds_reset_sync_in"),
+                (7, eem0, "dds_reset_sync_in", Misc("IOB=TRUE")),
```

Comment:
We didn't get this warning. Without it, we faced the symptoms of #1382 again. This just applies the same patch (bc3b55b1a83f824612ae78a843b2b0925c33e495), which seems to work for everyone. I'm not sure why it would pose a problem here, but maybe #1384 is a better place to discuss a proper solution?

Reply:
The warning shows up since that patch was applied. "Seems to work for everyone" can't be our benchmark for robustness, correctness and maintainability. Yes. Let's discuss it there.